FluentTerminal.App/Views/MainPage.xaml

```diff
@@ -48,8 +48,16 @@
                     <MenuFlyout>
                         <MenuFlyoutItem
                             Width="160"
-                            Command="{x:Bind ViewModel.AddTerminalCommand}"
-                            x:Uid="NewTab">
```

Comment:
Uid went missing, this is necessary for localization

Reply:
![image](https://user-images.githubusercontent.com/30559812/57468549-f4cf8100-7284-11e9-9e7c-d053fe91688d.png)
Also text is cut off